crates/oxc_linter/src/utils/react.rs

```diff
@@ -90,6 +90,10 @@ pub fn is_hidden_from_screen_reader(node: &JSXOpeningElement) -> bool {
     has_jsx_prop_lowercase(node, "aria-hidden").map_or(false, |v| match get_prop_value(v) {
         None => true,
         Some(JSXAttributeValue::StringLiteral(s)) if s.value == "true" => true,
```

Comment:
Even before the correction, it worked when a string was passed to aria-hidden, as shown below:

```jsx
// pass
<img alt='photo of cool person' aria-hidden='true' />
```